taichi/aot/graph_data.h

```diff
@@ -161,8 +162,16 @@ struct TI_DLL_EXPORT CompiledGraph {
   RuntimeContext ctx_;
 
   void run(const std::unordered_map<std::string, IValue> &args) const;
+  void jit_run(const CompileConfig &compile_config,
```

Comment:
@PGZXB do we want to expose a `jit_run` to external users? Say if a user call `jit_run` twice they will accidentally compile the kernels twice right? Maybe hiding the jit logic inside `run` will help?